lib/spack/spack/test/conftest.py

```diff
@@ -63,6 +64,59 @@ def no_stdin_duplication(monkeypatch):
                         lambda x: StringIO())
 
 
+@pytest.fixture()
+def spec_from_dict():
```

Comment:
:+1: I put it here because I thought there were no cases in which we could use it in core.